Symfony/src/AppBundle/Entity/User.php

```diff
@@ -32,6 +38,11 @@ class User implements UserInterface, \Serializable
      * @var string
      *
      * @ORM\Column(name="email", type="string", length=255, unique=true)
+     *
+     * @Assert\Length(
```

Comment:
Hi John, there is already an existing check when the user registers. Is a double check necessary?